src/main/resources/anniv6Resources/localization/eng/GremlinTown/UIstrings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "${ModID}:GremlinTown": {
+    "TEXT": [
+      "Gremlin Town",
+      "A section of the city populated by gremlins.  A more dangerous zone with stronger rewards."
```

Comment:
I just merged in code that will show the author for each zone. You should add an entry here with the name you'd like to be credited by. You can pull and look at other zones for examples.